functests/3_performance_status/status.go

```diff
@@ -103,6 +115,43 @@ var _ = Describe("Status testing of performance profile", func() {
 
 			mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
 		})
+
+		It("[test_id:40402] Tuned profile status tied to Performance Profile", func() {
+			// During this test we're creating additional synthetic tuned CR by invoking the createrBadTuned function.
+			// This synthetic tuned will look for a tuned profile which doesn't exist.
+			// This tuned CR will be applied on the profiles.tuned.openshift.io CR (there is such profile per node)
+			// which is associate to the node object with the same name.
+			// The connection between the node object and the tuned object is via the MachineConfigLables, worker-cnf in our case.
+			ns := "openshift-cluster-node-tuning-operator"
+			tunedName := "openshift-cause-tuned-failure"
+
+			// Make sure to clean badTuned object even if the It threw an assertion
+			clean = func() error {
```

Comment:
I'm wondering if an `eventually` block at the end of the function that does the cleanup would be better here.
having after each means something more inclusive (maybe also for future test)

Reply:
Not sure I understand what you mean.
You suggest to add ```eventually``` block at the end of the ```clean``` function. 
The ```clean()``` call will still be invoked by the ```AfterEach``` func

Reply:
adding eventually at the end of the test with the clean code

Reply:
If the test failed in the middle, for example the condition won't be updated properly [here](https://github.com/openshift-kni/performance-addon-operators/pull/598/files#diff-46dd9c2fd680952d6ae2734f9f4b9d42734d386f897dcb79cf030f17eae6f8f4R149), the test will exist without invoking any additional function besides ```AfterEach```
IOW if the ```eventually``` which contains the clean code, is after the line in which the test failed, it won't work.

Reply:
ok . works for me